pkgs/data/fonts/linearicons-free/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, fetchzip, stdenvNoCC }:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "linearicons-free";
+  version = "1.0.0";
+
+  src = fetchzip {
+    url = "https://cdn.linearicons.com/free/${finalAttrs.version}/Linearicons-Free-v${finalAttrs.version}.zip";
+    hash = "sha256-0Gb0CRdgzSnpeN+x8TrH5TCrAA57+jsBWZ4FgJ8cm08=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D -m444 -t $out/share/fonts/truetype $src/Desktop\ Font/Linearicons-Free.ttf
+
+    runHook postInstall
+  '';
+
+  meta = {
```

Comment:
How about adding the `platforms` attribute?

Reply:
done!